pkg/bgpv1/manager/manager.go

```diff
@@ -47,10 +49,43 @@ type LocalInstanceMap map[string]*instance.BGPInstance
 
 type bgpRouterManagerParams struct {
 	cell.In
-	Logger        logrus.FieldLogger
-	ConfigMode    *mode.ConfigMode
-	Reconcilers   []reconciler.ConfigReconciler   `group:"bgp-config-reconciler"`
-	ReconcilersV2 []reconcilerv2.ConfigReconciler `group:"bgp-config-reconciler-v2"`
+	Logger           logrus.FieldLogger
+	JobGroup         job.Group
+	ConfigMode       *mode.ConfigMode
+	Reconcilers      []reconciler.ConfigReconciler   `group:"bgp-config-reconciler"`
+	ReconcilersV2    []reconcilerv2.ConfigReconciler `group:"bgp-config-reconciler-v2"`
+	StateReconcilers []reconcilerv2.StateReconciler  `group:"bgp-state-reconciler"`
```

Comment:
bgp-state-reconciler => bgp-state-reconciler-v2 for consistency? This also tells readers that the state reconciliation is only for BGPv2.